SentryAot.slnf

```diff
@@ -0,0 +1,12 @@
+{
+  "solution": {
+    "path": "Sentry.Aot.sln",
+    "projects": [
+      "samples\\Sentry.Samples.Console.Basic\\Sentry.Samples.Console.Basic.csproj",
+      "src\\Sentry\\Sentry.csproj",
```

Comment:
Is there already a TODO/issue somewhere that we would need to combine this into the main solution nuget package? Hopefully using the following works: 
* https://learn.microsoft.com/en-us/nuget/create-packages/supporting-multiple-target-frameworks 
* and https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json#L1043

Reply:
I'm not sure I understand. `Sentry.Aot.sln` is a copy of `Sentry.sln`. The only reason it exists is so that we can  do things like this:
```
<PropertyGroup Condition="'$(SolutionName)' == 'Sentry.Aot'">
```

The only other ways I could think of to do that would be:
- We always build/test AOT for some specific version of .net (e.g. `net8.0`)
  - Not ideal, as then we couldn't also have managed tests against the `net8.0` 
- Pass in a flag from the command line or an environment variable
  - Would work OK in CI but not ideal for developers in an IDE
  
Do you think we could use some combination of the things you've referenced above to achieve the same, more simply?

Reply:
I think we're talking about orthogonal problems. What you're saying is all true and I generally agree with the solution. What I've meant by the comment is that we'll need a followup to make sure the sentry nuget package includes sentry.dll for both solutions. I think we'll need to do this manually (i.e. have a script)

Reply:
Ah I see... because of all the #ifdef stuff we've got going on, we can't have a single package that is both trimmable and not trimmable. Do you know how NuGet would know which dll to select?

Reply:
I think nuget would pick the right one if we structure the libraries correctly, i.e. put it in `/runtimes/aot/net8.0` or somewhat like that. But that's just my best guess from the documentation, I haven't tested it yet.

Reply:
Created https://github.com/getsentry/sentry-dotnet/issues/2759 for this.